verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -171,6 +171,9 @@ class KytheFactsExtractor {
   // Extracts Kythe facts from class instance node and return its VName.
   VName ExtractClassInstances(const IndexingFactNode& class_instance_fact_node);
 
+  // Creates a new temporary scope for if conditions and loops.
```

Comment:
An extractor should not be creating anything, even if temporary.  The unnamed scopes created by the indexer should persist.